RELEASING.adoc

```diff
@@ -1,105 +1,75 @@
 = Releasing Kiali UI
 
-This document covers the specifics of releasing Kiali UI, and is meant as a
-guide for Kiali release managers.
-
-The main document covering core can be found link:https://github.com/kiali/kiali/blob/master/RELEASING.adoc[here].
-
-== Requirements
-
-You must have write permissions to the public GitHub repository in order to be
-able to push the tags, but that's pretty much it. Most of our release process
-is done automatically by Travis.
-
-This document also uses 1.0.0 as the new version, just as an example.
-
-== Preparing for a new version
-
-After a release, the first step for a new "batch" of modifications is to update
-link:package.json[] with the new version, and creating a PR for it:
-
-[source, diff]
-----
-diff --git a/package.json b/package.json
-index 7f699b1..08bc0a8 100644
---- a/package.json
-+++ b/package.json
-@@ -1,6 +1,6 @@
- {
-   "name": "@kiali/kiali-ui",
--  "version": "0.5.1",
-+  "version": "1.0.0",
-   "bugs": {
-     "url": "https://issues.jboss.org/projects/KIALI/issues/"
-   },
-----
-
-This is to signal to everyone that everything that goes into master from that
-point forward belongs to that version. Note that this is supposed to be the
-same version that is going to be released for the core, so they are in sync.
-
-The versioning scheme that we are following is similar to
-link:http://semver.org[semver], but not as strict, given both that we are not
-1.0 yet and that we are the end product, in the sense that nothing should
-depend on us to do anything. For now, and that might change in the future, our
-API is private and we make no guarantees that compatibility is going to be
-maintaned.
```

Comment:
I think this is useful to have, unless is specified in the main release process, then i'm ok with removing it here.

Reply:
I restored back all that block. It is useful info, indeed.